nautobot/wireless/choices.py

```diff
@@ -0,0 +1,73 @@
+from nautobot.core.choices import ChoiceSet
+
+
+class RadioFrequencyChoices(ChoiceSet):
+    FREQUENCY_24G = "2.4 GHz"
+    FREQUENCY_5G = "5GHz"
+    FREQUENCY_6G = "6GHz"
+
+    CHOICES = (
+        (FREQUENCY_24G, "2.4 GHz"),
+        (FREQUENCY_5G, "5 GHz"),
+        (FREQUENCY_6G, "6 GHz"),
+    )
+
+
+class RadioStandardChoices(ChoiceSet):
+    A = "802.11a"
+    B = "802.11b"
+    G = "802.11g"
+    N = "802.11n"
+    AC = "802.11ac"
+    AX = "802.11ax"
+    BE = "802.11be"
+
+    CHOICES = (
+        (A, "802.11a"),
+        (B, "802.11b"),
+        (G, "802.11g"),
+        (N, "802.11n"),
+        (AC, "802.11ac"),
+        (AX, "802.11ax"),
+        (BE, "802.11be"),
+    )
+
+
+class WirelessDeploymentModeChoices(ChoiceSet):
+    CENTRAL = "Central (tunnelMode(controller managed))"
+    FABRIC = "Fabric"
+    STANDALONE = "Standalone (Autonomous)"
+    LOCAL = "Local (Flex)"
+    MESH = "Mesh"
+    BRIDGE = "Bridge"
```

Comment:
I'm not saying that we have time to do this but we *should* look into how all of the major wireless vendors (Cisco, Aruba, Huawei, Ubiquiti) refer to these modes and use vendor-agnostic choices.

Reply:
This is what Architecture provided. This could be easily changed in the future.